charts/rstudio-connect/values.yaml

```diff
@@ -261,6 +261,33 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
+# -- Settings for the Chronicle Agent sidecar container
+chronicleAgent:
```

Comment:
I would have expected `environment` to be an option here. (This is related to the perhaps-poorly-named "environment" option for the agent: https://docs.posit.co/chronicle/appendix/library/advanced-agent.html#environment).

Maybe this is defined somewhere else?

Reply:
I think since this comment I added `env` for this purpose.

Reply:
I don't think this is quite the same thing.

We have a gcfg variable named `Agent.Environment`. This is optional, but intended to allow our users to "tag" metrics with the deployment environment name (e.g., "test" vs "prod").

We expect most small customers won't care about this, but most large customers will heavily rely on this option.

I would expect that where we have `serverAddress` and `connectApiKey`, that we would have something like `chronicleEnvironment` (or perhaps some better name???). Conceptually, these variables are all the same kind of thing (configuration for the agent).

Reply:
Ah! So this is a difference between [environment variable over-rides](https://docs.posit.co/chronicle/appendix/library/advanced-agent.html#environment-variables) and the `Agent.Environment` configuration value. Thanks for the clarification.

Reply:
I created `agentEnvironment` for this purpose.